packages/ai-jsx/src/core/core.ts

```diff
@@ -428,7 +433,7 @@ export function createRenderContext(opts?: { logger?: LogImplementation }) {
   });
 }
 
-function createRenderContextInternal(render: StreamRenderer, userContext: Record<symbol, any>): RenderContext {
+function createRenderContextInternal(renderStream: StreamRenderer, userContext: Record<symbol, any>): RenderContext {
```

Comment:
This code is pretty tough to parse because the word `render` is overused. I changed this param name to match the name of the value that's passed into it, but there's more we can do here.